sdc/io/csv_ext.py

```diff
@@ -388,10 +388,10 @@ def _sanitize_varname(varname):
 def to_varname(string):
     """Converts string to correct Python variable name.
     Replaces unavailable symbols with _ and insert _ if string starts with digit.
+    In case where many underscores in a row, replace by 1 underscore.
```

Comment:
I would propose to simplify this line: "Remove excess underscores from the string."